charts/README.md

```diff
@@ -0,0 +1,250 @@
+[![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/awslabs/kubeflow-manifests/issues)
```

Comment:
please remove all the READMEs under this folder and any subdirectories. Docs will be added to website as discussed